resources/views/admin/users/edit.blade.php

```diff
@@ -430,6 +430,15 @@
           },
           profileUpdate($event) {
             this.resetErrors();
+            if (@json($enabled2FA) &&  this.global2FAEnabled.length === 0) {
+              // User has not enabled two-factor authentication correctly
+              ProcessMaker.alert(
+                this.$t('The Two Step Authentication Method has not been set. Please contact your administrator.'),
```

Comment:
Missing translation, not found in `resources/lang/en.json`
```
The Two Step Authentication Method has not been set. Please contact your administrator.
```

Reply:
the label exists.